### ISSUE TITLE
touchstart event does not contain proper values in changedTouches property on android devices 

### ISSUE THREAD

**Issue by Bamyan** [DevExpress/testcafe#2856]
### Are you requesting a feature or reporting a bug?
bug

### What is the current behavior?

It is not adding touches into `event.changedTouches` array. it is always empty

### What is the expected behavior?
Should contain touches in `event.changedTouches` array as i does in non-testcafe environment

### How would you reproduce the current behavior (if this is a bug)?
call `t.click` method on element with onTouchStart handler and check the changedTouches property

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

it is an Angular code to simplify the example:

Test code

```TS:

onTouchStart(e:TouchEvent) {
    this.xyz = e.changedTouches.length; //In real application it is 1, in testcafe it is 0
}

```

```HTML:
<button id="test" (touchstart)="onTouchStart($event)">test</button>
{{xyz}}
```

### Specify your

* operating system: Android (Chrome)
* testcafe version: 0.22.0
* node.js version: 8.9.4

**Comment by AndreyBelym** [comment]
Hi @Bamyan, sorry for the delayed response! The `changedTouches` property must be emulated on Android devices, we have a unit test for it: [testcafe-hammerhead/test/client/fixtures/sandbox/event/event-simulator-test.js](https://github.com/DevExpress/testcafe-hammerhead/blob/9ff9850765dfc27b00cafabeda552bed26192f2a/test/client/fixtures/sandbox/event/event-simulator-test.js#L268). Could you please provide an example page where the problem can be reproduced?

**Comment by miherlosev** [comment]
I will close the issue because no activity has been encountered for a long time.
Feel free to reopen the issue if the problem persists and you can provide an example to reproduce it.

**Comment by Bamyan** [comment]
Hi. @miherlosev @AndreyBelym , Sorry for the Late Reply, you can see the problem with the following test on example page.
```
fixture `click on checkbox`
    .page(`https://v6.material.angular.io/components/checkbox/overview`);

test('Should click on checkbox', async () => {
    await t.click(Selector('mat-checkbox'));
});
```
ERROR:
`Uncaught TypeError: Cannot read property 'clientX' of undefined`

**Comment by Bamyan** [comment]
Hi. @miherlosev @AndreyBelym , Sorry for the Late Reply, you can see the problem with the following test on example page.

```
fixture `click on checkbox`
    .page(`https://v6.material.angular.io/components/checkbox/overview`);

test('Should click on checkbox', async () => {
    await t.click(Selector('mat-checkbox'));
});
```
ERROR:
`Uncaught TypeError: Cannot read property 'clientX' of undefined`

**Comment by miherlosev** [comment]
I've tried to reproduce the problem with `testcafe@0.22.1-alpha.3` and TestCafe works fine.
Try to run tests with the latest TestCafe version (`testcafe@0.22.1-alpha.3`).
If the problem persists, please specify the Google Chrome version in which you run your tests.

![image](https://user-images.githubusercontent.com/4133518/47296910-606d3c00-d61c-11e8-9ee3-147117f36c56.png)

**Comment by miherlosev** [comment]
Thank for your detailed report. I've reproduced the problem.

**Comment by AndreyBelym** [comment]
It can be reproduced with our Drag functional tests & Chrome 70. The latest Chrome version on BrowserStack is 68, that's why we haven't discovered this before.

**Comment by jl-ib** [comment]
Hi, any updates on this? @AndreyBelym

**Comment by AlexKamaev** [comment]
The issue occurs in a core `testcafe-hammerhead` module. It is fixed, but it will be available after a new version of `testcafe-hammerhead` is published and used in testcafe. If you want to test the fix, you can pull the latest `testcafe-hammerhead` branch and add it to your project manually.

**Comment by AndreyBelym** [comment]
Fixed in the `testcafe@0.23.1-alpha.4`.
